pkg/scheduler/core/spreadconstraint/group_clusters.go

```diff
@@ -128,6 +128,46 @@ func groupClustersIgnoringTopology(
 	return groupClustersInfo
 }
 
+const weightUnit int64 = 1000
+
+func (info *GroupClustersInfo) calcGroupScore(
+	clusters []ClusterDetailInfo,
+	rbSpec *workv1alpha2.ResourceBindingSpec,
+	minGroups int) int64 {
+	// if the replica scheduling type is divided, the score is calculated by followed.
+	int32MinGroups := int32(minGroups)
+	restReplica := rbSpec.Replicas % int32MinGroups
+	replica := rbSpec.Replicas - restReplica
+	targetReplica := int64(replica/int32MinGroups + 1)
+
+	// clusters have been sorted by cluster.Score descending,
+	// and if the cluster.Score is the same, the cluster.availableReplica is ascending.
+	var sumAvailableReplica int64
+	var sumScore int64
+	var validClusters int64
+	for _, cluster := range clusters {
+		sumAvailableReplica += cluster.AvailableReplicas
+		sumScore += cluster.Score
+		validClusters++
+		if sumAvailableReplica >= targetReplica {
```

Comment:
a corner case here: 
targetReplica is `10`
group is `[{"availableReplicas": 10000, "score": 100}, {"availableReplicas": 10000, "score": 100}, {"availableReplicas": 10000, "score": 100}]`
sumAvailableReplica only adds the `AvailableReplicas` of a cluster, that is, this will cause the algorithm to degenerate into a situation where only one cluster score is calculated.
### Improvement measures
always take `minGroups` number of clusters.